blog/Konferencja Tłumaczy 2023 - relacja/index.md

```diff
@@ -0,0 +1,317 @@
+---
+title:
+  'Konferencja Tłumaczy 2023 – relacja'
+authors: jkula
+date: '2023-10-19'
+tags:
+  - 'news'
+  - 'z-kraju'
+  - 'konferencje'
+  - 'społecznosc'
+coverImage: 'KT2023-logo.jpg'
+---
+
+Konferencja Tłumaczy 2023 już za nami. Ale poniżej czeka na Was relacja. Zapraszamy do lektury!
+
+<!--truncate-->
+
+## W dniu św. Hieronima, patrona tłumaczy
```

Comment:
"Na świętego Hieronima jest pogoda lub jej ni ma" 😉 

Takie przysłowie mi zawsze wskakuje do głowy jak widzę to imię. Nie jest to rzeczowy komentarz do PRa, tylko taki wygłup 😅